st2client/tests/unit/test_auth.py

```diff
@@ -39,6 +41,108 @@
 }
 
 
+class TestLogin(base.BaseCLITestCase):
+
+    def __init__(self, *args, **kwargs):
+        super(TestLogin, self).__init__(*args, **kwargs)
+        self.parser = argparse.ArgumentParser()
+        self.parser.add_argument('-t', '--token', dest='token')
+        self.parser.add_argument('--api-key', dest='api_key')
+        self.shell = shell.Shell()
+
+    def setUp(self):
```

Comment:
setUp and tearDown aren't needed here since we are not overriding a default behavior.